### PR TITLE
Replaced "sanitize" with "no-sanitize". 

### LINKED ISSUE #40: The marked-element in insecure by default
### Description

The marked-element has sanitize set to false by default, meaning that users of it are vulnerable to XSS unless it occurs to them to add the word sanitize.

It would be more sensible to add a noSanitize property (that defaults to false), and people that really trust their markdown can set it.

It will be obvious to people who need no-sanitize that something's wrong, and most likely no harm will come to them as a result, unlike the current situation.
### Expected outcome

That Polymer projects are not vulnerable to XSS
### Actual outcome

Users of the marked-element need to ensure they pass the sanitize attribute to their tags, or they'll be vulnerable to XSS when using user-generated markdown.

### PR DESCRIPTION
**Pull request by TomCools**
Made santize for all content the default and added a "no-sanitize" boolean property so you can disable this default.

It was mentioned in an Issues, but I decided to just go ahead and create the MR for it.

Fixes #40

### CODE REVIEW

**Comment by TomCools** [comment]
I signed it!

**Comment by TomCools** [comment]
I signed it!

**Comment by e111077** [comment]
Heya, thanks for the PR, my reservations about this are:

1. This is inherently a breaking change
2. Goes against Google style guidelines where booleans should not be named negatively (noSanitize / dontSanitize vs sanitize)

Perhaps unsanitized is a better name.

Since this is a breaking change I will have to run more-extensive internal integration tests to see the extent of the breakages and determine if it's fixable and does not break too many targets.

File: marked-element.html [hunk 1]
```diff
@@ -142,9 +142,9 @@
         value: null
       },
       /**
-       * Sanitize the output. Ignore any HTML that has been input.
+       * Do not sanitize the output. Default ignores any HTML that has been input.
```
Comment (MajorBreakfast) [inline marked-element.html:145]: Better phrasing:

Disable sanitization. Default: Ignore any HTML tags contained within the input.

**Comment by MajorBreakfast** [comment]
About the naming: I just wanted to mention that it's not uncommon that properties start with "no". 
https://www.webcomponents.org/element/PolymerElements/paper-dialog/elements/paper-dialog
https://www.webcomponents.org/element/PolymerElements/paper-button/elements/paper-button
However, "unsanitized" sounds good.

I'm generally a big fan of the "safe by default" mentality. However, I do not think that it is wise to merge this. This will definitely break things. It'd be good if there was a deprecation warning to raise awareness first and to let `sanitize` and `no-sanitize` exist in parallel for a about a year or so.

**Comment by e111077** [comment]
I can confirm that this is causing significant breakages across Google.

In terms of naming; I _swear_ I saw this in one of our style guides but I can't seem to find it. `noSanitize` is okay

**Comment by e111077** [comment]
Overall, due to the breaking changes, I'm going to have to close this PR. `noSanitize` living parallel to `sanitize` is going to cause a lot of trouble and historically users don't tend to listen to warnings which will just break a lot of users.

**Pull request closed without merging**